### DISCUSSION TITLE
Issue Access denied finding property "vendor.camera.aux.packagelist"

### DISCUSSION THREAD

**Discussion by hommalex** [capawesome-team/capacitor-mlkit#131] [General]
I have an issue with my set on Android. Can anybody help.

I'm pretty much sure it's nothing to do with the barcode scanner itself.

I have permission granted:
![image](https://github.com/capawesome-team/capacitor-mlkit/assets/12752800/d806a9df-b2ee-4c9d-97a3-6b7792b1111f)

I have manifest updated:
![image](https://github.com/capawesome-team/capacitor-mlkit/assets/12752800/fa88f692-96e0-4fef-be58-ac1c1556484a)

And I'm still getting this error:
Access denied finding property "vendor.camera.aux.packagelist"

The camera is not opening.

I have tried in 3 different devices.
The all project is building fine. And all packages just been updated.

Here is the package.json:
![image](https://github.com/capawesome-team/capacitor-mlkit/assets/12752800/de753990-0674-4ee5-9ca7-24158024624a)

Anybody can explain what's going on. Please?

**Comment by robingenz** [comment]
Please provide a [Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example) using [this template](https://github.com/capawesome-team/.capacitor-app) in a public GitHub repository so I can debug the issue.

**Comment by hommalex** [comment]
Turns out the new version of @capacitor/android 5.7.0 and @capacitor/core 5.7.0 Are not camera friendly. I just had to downgrade those.
